Mention SubViewport.render_target_clear_mode property is intended for 2D usage

Yetizone opened the pull request:
Fixes https://github.com/godotengine/godot/issues/33831

<!--
Pull requests should always be made for the `master` branch first, as that's
where development happens and the source of all future stable release branches.

Relevant fixes are cherry-picked for stable branches as needed.

Do not create a pull request for stable branches unless the change is already
available in the `master` branch and it cannot be easily cherry-picked.
Alternatively, if the change is only relevant for that branch (e.g. rendering
fixes for the 3.2 branch).
-->

Yetizone committed:
Mention SubViewport.render_target_clear_mode property is intended for 2D usage.

NathanLovato commented on doc/classes/SubViewport.xml:
```diff
@@ -10,7 +10,7 @@
 	<members>
 		<member name="render_target_clear_mode" type="int" setter="set_clear_mode" getter="get_clear_mode" enum="SubViewport.ClearMode" default="0">
-			The clear mode when the sub-viewport is used as a render target.
+			The clear mode when the sub-viewport is used as a render target. Note this property is intended for 2D usage.
```
```suggestion
			The clear mode when viewport used as a render target.
			[b]Note:[/b] this property is intended for 2D usage.
```

Yetizone replied:
I've noticed that in your comment here you're also suggesting to change the first sentence, i.e. replace "the sub-viewport" with "viewport" and remove "is". 
Just wanted to double check if you're sure about the changes or if they were perhaps unintentional :sweat_smile:

NathanLovato replied:
It was unintentional, yes. I'm fixing it

NathanLovato reviewed (changes requested):
Suggesting a formatting change for consistency with the rest of the class ref

NathanLovato reviewed (changes requested):
You added some commits to your PR inadvertently

akien-mga commented:
Needs a rebase as I exceptionally force-pushed a rebase of the `master` branch to fix the unwanted "wip1" commit shown above.

Then the same style fixes as in #41713 can be done too.

NathanLovato commented:
I edited the suggested change to reflect the style guide

Yetizone commented:
Okay, rebased and adjusted style. 

Thanks for letting me know what happened, I was a bit confused when I saw the PR suddenly had 1900+ additions :sweat_smile:

akien-mga commented:
Thanks!
